src/listmode_buildblock/LmToProjData.cxx

```diff
@@ -647,40 +649,40 @@ process_data()
 		 // a record can never be both timing and coincidence event
 		 // and there might be a scanner around that has them both combined.
 		 if (record.is_event())
-		   {
+           {
```

Comment:
quite a few white-space changes below, and unfortunately most seem wrong. Can you revert them? (I think your editor inserts tabs. Please change it not to.)